.gitignore

```diff
@@ -9,4 +9,4 @@ tools/geckodriver
 tools/chromedriver
 
 geckodriver.log
-.DS_Store
+.DS_Store
```

Comment:
Can you remove the changes to this file from the PR?

Reply:
Yeah sure.